mathesar/tests/integration/utils/locators.py

```diff
@@ -0,0 +1,9 @@
+from playwright.sync_api import Page
+
+
+def get_tables_list(page: Page):
+    return page.locator("#sidebar li[aria-level='1']:has(button:has-text('Tables')) ul")
```

Comment:
These are much better suited as properties of a Page object model as they belong to a particular page(tables list page).

Reply:
How would you feel about making a PR with these changes after merging this PR?